File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extra) {
        int max = Integer.MIN_VALUE;
        for(int i : candies) max = Math.max(max,i);
        List<Boolean> list = new ArrayList<>();
        for(int i : candies) list.add(i+extra >=max);
        return list;
    }
}

File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extra) {
    let max = -Infinity;
    for(let i of candies) max = Math.max(max,i);    

    let list = [];
    for(let i of candies){
        list.push(i+extra >= max);
    }
    return list;
};